### PR TITLE
Use juvix-installer in CodeSpaces, install vamp-ir

### LINKED ISSUE #2113: Codespace build is failing

### PR DESCRIPTION
**Pull request by vrom911**
- Closes #2113
- Install `vamp-ir`.

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -62,13 +64,12 @@ RUN sudo bash -c 'curl https://get.wasmer.io -sSfL | sudo WASMER_DIR=/home/vscod
...
 
-RUN sudo bash -c 'curl -s https://api.github.com/repos/anoma/juvix/releases/latest \
-      | grep -a "browser_download_url.*linux" \
-      | cut -d : -f 2,3 \
-      | tr -d \" \
-      | wget --output-document juvix.zip -qi - \
-      && unzip juvix.zip \
-      && mv juvix /home/vscode/.local/bin/juvix'
+
+RUN sudo bash -c 'curl --proto "=https" --tlsv1.2 -sSf https://raw.githubusercontent.com/anoma/juvix-installer/main/juvix-installer.sh \
```
Comment (jonaprieto) [inline .devcontainer/Dockerfile:68]:
```suggestion
RUN sudo bash -c 'curl --proto "=https" --tlsv1.2 -sSf https://get.juvix.org \
```

File: .devcontainer/Dockerfile [hunk 2]
```diff
@@ -62,13 +64,12 @@ RUN sudo bash -c 'curl https://get.wasmer.io -sSfL | sudo WASMER_DIR=/home/vscod
...
+      | sudo JUVIX_INSTALLER_INSTALL_VAMPIR_YES=true JUVIX_INSTALLER_ASSUME_YES=true sh \
+      && sudo mv /root/.local/bin/juvix /home/vscode/.local/bin/ \
+      && sudo mv /root/.local/bin/vamp-ir /home/vscode/.local/bin/'
```
Comment (jonaprieto) [inline .devcontainer/Dockerfile:71]: Check if you can use `XDG_BIN_HOME=/home/vscode/.local/bin` to not use `mv` here.

**Review by jonaprieto — CHANGES_REQUESTED** [review]

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (paulcadman) [reply .devcontainer/Dockerfile:68]: With this suggestion, you need to include the `-L` option to curl because https://get.juvix.org is a redirect.

**Comment by vrom911** [comment]
All done now!

**Review by jonaprieto — APPROVED** [review]

**Pull request merged**